### PR TITLE
Ess-3110 Not all fields are mapped when getting a campaign

### PR DESCRIPTION
**Pull request by pamela-joslyn-4ss**
## Description
Map all fields on get_campaign
Also fixed bug ESS_3038 get_swimops_campaign does not return gracefully if no operations

## Checklist
- [x] PR title is descriptive and fit for injection into release notes (see tips below)
- [x] Correct label(s) are used

### CODE REVIEW

**Comment by bjorn-einar-bjartnes-4ss** [comment]
We have to fix
- styling issues
- Make sure with tests it handles non-swim as well as swim campaigns.

**Comment by hakon-molven-4ss** [comment]
> We have to fix
> 
> * styling issues
> * Make sure with tests it handles non-swim as well as swim campaigns.

I snuck in and fixed the style by running `black .` and `isort . --profile black` with the latest `black` and `isort` installed. Sneaking back out...

File: .vscode/extensions.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "recommendations": [
+        "ms-python.black-formatter"
```
Comment (bjorn-einar-bjartnes-4ss) [inline .vscode/extensions.json:3]: It is next to impossible to work with black without some formatter in VS Code...

File: tox.ini [hunk 2]
```diff
@@ -6,7 +6,7 @@ envlist =
 
 [testenv]
 commands =
-    pytest --basetemp={envtmpdir} --cov=fourinsight.campaigns --cov-report html:cov_html-{envname} --junitxml=junit-{envname}.xml ./tests
+    pytest --basetemp={envtmpdir} --cov=fourinsight.campaigns --cov-report html:cov_html-{envname} --junitxml=junit-{envname}.xml ./tests -vv
```
Comment (bjorn-einar-bjartnes-4ss) [inline tox.ini:9]: This prints the full and not truncated diff in tests.

File: tests/test_api.py [hunk 3]
```diff
@@ -278,6 +287,15 @@ def test_get_campaign_camelcase(
             "Main Data Provider": "4Subsea",
             "Start Date": "2017-04-08T00:00:00+00:00",
             "End Date": "2017-05-13T00:00:00+00:00",
+            "GeoTrack Location": "10#21",
```
Comment (bjorn-einar-bjartnes-4ss) [inline tests/test_api.py:290]: I don't know why it needs all these, I assume they have been added to the API without the package having been built. Perhaps we should run these tests when we deploy 4insight, or at least at some intervals to stay up to date on changes.

File: .vscode/extensions.json (hunk 1 above)
Comment (bjorn-einar-bjartnes-4ss) [reply .vscode/extensions.json:3]: I think this also needs to run on save somehow. Or be a task in vs code.

File: fourinsight/campaigns/api.py [hunk 4]
```diff
@@ -194,8 +194,11 @@ def get_campaign(self, campaign_id):
         """
         response_map = {
             ("id", "CampaignID"): None,
-            ("projectnumber", "Project Number"): None,
```
Comment (bjorn-einar-bjartnes-4ss) [inline fourinsight/campaigns/api.py:197]: move this around so it does not look like a removal.

**Review by bjorn-einar-bjartnes-4ss — APPROVED** [review]

**Pull request merged**